test/lib/jdk/test/lib/process/OutputAnalyzer.java

```diff
@@ -502,7 +502,7 @@ public OutputAnalyzer shouldNotHaveExitValue(int notExpectedExitValue) {
         if (getExitValue() == notExpectedExitValue) {
             reportDiagnosticSummary();
             throw new RuntimeException("Unexpected to get exit value of ["
-                    + notExpectedExitValue + "]\n");
+                    + notExpectedExitValue + "], exit value is: [" + getExitValue() + "]");
```

Comment:
This seems redundant to me. "Unexpected to get exit value of [notExpectedExitValue]." should suffice?

Reply:
I agree this doesn't need changing as there is only one value to report.

Reply:
Right, a bit overzealous on the expected vs actual.